### DISCUSSION TITLE
2024/02/08/LIS_nlogn_plus/

### DISCUSSION THREAD

**Discussion by giscus** [MinecraftDream12345/MinecraftDream12345.github.io#8] [Announcements]
# 2024/02/08/LIS_nlogn_plus/

大秦为您打开题目传送门（3/1 题目大意题目的表述十分清楚，他就是让你求字典序最小的最长上升子序列，如果不存在输出 orz 。 注：在某人jdt1010的提示下发现不存在的情况是当 $n=0$ 的时候分值为两分多一点。 3/2 解题思路方法1：$dp[i]$ 表示以 $i$ 结尾的最长上升子序列长度我们阅读题目发现，这道题 $n$ 的数据范围非常大，传统的 $O(n^

https://minecraftdream12345.github.io/2024/02/08/LIS_nlogn_plus/

**Comment by FlyingYummyWolf** [comment]
%%%%%%%%%%%orz
